docs/ota.md

```diff
@@ -51,10 +61,10 @@ Notify PTA BO that the bus is starting a block.
 #### Example payload
 ```json
 {
-  "eventTimestamp": "2017-10-31T12:45:50Z",
-  "vehicleNumber": "12345",
-  "blockId": "1234:34",
-  "vehicleJourneyId": "35:ABC"
+   "eventTimestamp": "2019-05-04T15:03:10Z",
+   "vehicleNumber": "E2345678901234567",
+   "blockId": "3103",
+   "vehicleJourneyId": "294"
```

Comment:
Her aksepterer vi også IDen som blockID-Starttid
Der starttid må vere ISO 8601 date i lokal tidssone, altså CE(S)T.
Det burde nok også vere dokumentert

Reply:
Takk for input!